src/client/components/Analytics/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { Table } from '@instructure/ui-table';
+import axios from 'axios';
+import { ltikPromise } from '../../services/ltik';
+
+const Analytics = () => {
+  const [analytics, setAnalytics] = useState([]);
+
+  const getAnalytics = () => {
+    ltikPromise.then(ltik => {
+      axios.get(`/api/getanalytics?ltik=${ltik}`).then(res => {
+        setAnalytics(res.data);
+      });
+    });
+  };
+
+  useEffect(getAnalytics, []);
+  console.log(analytics);
```

Comment:
@finnzink Errant adding of console.log.

Please add this to the eslint config so it can be checked and we stop having this committed in the final pull request version? https://eslint.org/docs/rules/no-console